Codigo/Protocol.bas

```diff
@@ -19381,9 +19384,17 @@ End Sub
 
 Private Sub HandleDebugRequest(ByVal UserIndex As Integer)
 On Error GoTo HandleDebugRequest_Err:
-    
-    If (UserList(UserIndex).flags.Privilegios And (e_PlayerType.Admin Or e_PlayerType.Dios Or e_PlayerType.Consejero)) Then
-        Call WriteDebugLogResponse(UserIndex)
+    Dim debugType As Byte
+    Dim args() As String
+    debugType = Reader.ReadInt8()
+    If debugType = 1 Then
```

Comment:
if debugType is invalid value then call `exit sub`

Reply:
Ill rebase, this should not be on this mr